readmeAssets/logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="192" height="64" viewBox="0 0 192 64" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I can't really see the logo, it's very white. I believe we should make it darker.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
<img width="222" alt="Capture11" src="https://user-images.githubusercontent.com/92370865/174553566-f7c2bfd8-be44-4a44-a0bd-ef720da37ba8.PNG">
I don't think so. The logo renders fine but when open separately it's become very white